BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleThumbnailsHelper.m

```diff
@@ -41,6 +41,7 @@ - (instancetype)init
 
 - (void)storeThumbnailForItemWithID:(NSString *)itemID userID:(NSString *)userID thumbnail:(UIImage *)thumbnail
 {
+    if (thumbnail == nil) return;
```

Comment:
I'm surprised this is even called with `nil`. Is it intended to be? And if so, is it intended to be a noop?